pkg/kubelet/kubelet.go

```diff
@@ -1510,7 +1547,7 @@ func (kl *Kubelet) updatePods(u PodUpdate, podSyncTypes map[types.UID]metrics.Sy
 		kl.pods = u.Pods
 		kl.pods = filterHostPortConflicts(kl.pods)
 	case UPDATE:
-		glog.V(3).Infof("Update: Containers changed")
+		glog.Infof("Update: Containers changed")
```

Comment:
Yup - makes debugging much easier, and I couldn't find a way of passing -v=X to test-go.sh... I'll revert those when creating 'real' RP.